taskcluster/ci/startup-test/kind.yml

```diff
@@ -36,7 +36,7 @@ job-defaults:
     run-on-tasks-for: []
     attributes:
         build-type: nightly 
-        nightly: true
+        #nightly: true
```

Comment:
@JohanLorenzo is it fair game to leave it this way until it's fixed?

Reply:
Yup, it looks good to me!

Nit: let's add a code comment to explain why we remove this task off the nightly graph.